src/condor_tools/htcondor_cli/annex_create.py

```diff
@@ -956,14 +962,14 @@ def annex_name_exists(annex_name):
 
 
 def annex_create(logger, annex_name, **others):
-    if others.get("test") is not True:
+    if others.get("test") is None:
```

Comment:
```suggestion
    if not others.get("test"):
```
unless you care about distinguishing between `""` and `None`

Reply:
or in this case distinguishing between `0` and `None`?

Reply:
Correct; I do want to distinguish between those; the test flag must not be present at all.